.github/workflows/cd.yml

```diff
@@ -21,41 +38,12 @@ concurrency:
   cancel-in-progress: false
 
 jobs:
-  get-version:
```

Comment:
We don't need this anymore, because we can get the latest release version from the input `image_tag` when calling from the `release`, or it's getting [by the terraform itself when passing an empty one](https://github.com/WalletConnect/echo-server/blob/40e6d51954c798ca86649c1f3c5a1f646523a73c/terraform/main.tf#L8-L9).